ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -411,12 +411,12 @@ Deploy Kserve Model Via UI  #robocop: disable
     Click Button    Deploy
     Wait Until Page Does Not Contain Element    xpath=${KSERVE_MODAL_HEADER}   timeout=60s
 
-Select Kserve Serving Runtime
+Set Model Server Runtime
     [Documentation]    Opens the Runtime dropdown in the deploy modal for kserve models and select the given runtime
```

Comment:
The doc says "...for **kserve** models...". Shouldn't it be more generic (like in the title, which now it's not about KServe)

Reply:
updated